### DISCUSSION TITLE
Error while loading shared libraries: libssl.so.1.1

### DISCUSSION THREAD

**Discussion by mununki** [vercel/community#590] [Help] [answered]
I bump into the error below. I run the deployment several times per day basis usually. It was fine until today. I'm suspicious that the Vercel deployment env might be changed, but not sure.
```
/vercel/path0/node_modules/rescript-relay/rescript-relay-compiler.exe: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
--
10:02:41.896 | error Command failed with exit code 127.
10:02:41.896 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
10:02:41.918 | error Command failed with exit code 127.
10:02:41.918 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
10:02:41.933 | Error: Command "yarn build" exited with 127
```

Does anybody have the same issue?

**Comment by gudmundur** [comment, marked as answer]
Hi @mattdamon108. 👋🏻 

Thanks for raising this issue. We had multiple versions of OpenSSL installed in our build system and the download for one of them broke. I deployed a change yesterday that removed the broken package entirely, which caused the issue you are seeing. I'm working on a fix to this that will go out in the next few hours.

~I'll update this comment once the fix is out and hopefully that'll resolve the issue for you.~

**Update**: A fix was rolled out at 15:04:00 UTC. If the problem persists for you, please let us know. Again, thanks for raising the issue and the patience here. 🥇

**Reply by mununki** [reply]
@gudmundur I ran a new deployment and checked the deployment seems okay now. Thank you for your prompt action.
